src/main/java/com/ratifire/devrate/service/UserService.java

```diff
@@ -458,9 +455,9 @@ public void createBookmark(long userId, BookmarkDto bookmarkDto) {
    * @param userId the ID of the user to associate the interview summaries with
    * @return A list of InterviewSummaryDto objects.
    */
-  public List<InterviewSummaryDto> getInterviewSummariesByUserId(long userId) {
+  public List<InterviewHistoryDto> getInterviewSummariesByUserId(long userId) {
     User user = findById(userId);
-    return interviewSummaryMapper.toDto(user.getInterviewSummaries());
+    return interviewHistoryDataMapper.toDto(user.getInterviewHistories());
   }
```

Comment:
I think we need to move this method to the `InterviewHistoryService` and remove all mentions about `InterviewHistoryService` and `InterviewHistoryRepository` and mapper from the UserService